<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportDialog</class>
 <widget class="QDialog" name="ImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import data</string>
  </property>
  <property name="windowIcon">
   <iconset theme="document-open">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="mainLayout">
   <item row="1" column="1">
    <widget class="QLineEdit" name="fseparatorEdit">
     <property name="placeholderText">
      <string>Field separator (default &quot;;&quot;)</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::West</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="docsTab">
      <attribute name="icon">
       <iconset theme="document">
        <normaloff>.</normaloff>.</iconset>
      </attribute>
      <attribute name="title">
       <string>Import documents</string>
      </attribute>
      <layout class="QGridLayout" name="docsTabLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="docNameLabel">
         <property name="text">
          <string>Document name index</string>
         </property>
         <property name="buddy">
          <cstring>docNameSpin</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="docsLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="documentEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Path to documents list</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="documentButton">
           <property name="text">
            <string>Open</string>
           </property>
           <property name="icon">
            <iconset theme="document-open">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="docPrecinctLabel">
         <property name="text">
          <string>Precinct index</string>
         </property>
         <property name="buddy">
          <cstring>docPrecinctSpin</cstring>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="docCommentSpin">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="docCommunitySpin">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="docCommentLabel">
         <property name="text">
          <string>Comments index</string>
         </property>
         <property name="buddy">
          <cstring>docCommentSpin</cstring>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QSpinBox" name="docOwnerSpin">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="docCommunityLabel">
         <property name="text">
          <string>Community index</string>
         </property>
         <property name="buddy">
          <cstring>docCommunitySpin</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="docLootSpin">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="docNameSpin">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="docLotLabel">
         <property name="text">
          <string>Lots index</string>
         </property>
         <property name="buddy">
          <cstring>docLootSpin</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="docPrecinctSpin">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="docOwnerLabel">
         <property name="text">
          <string>Owners start index</string>
         </property>
         <property name="buddy">
          <cstring>docOwnerSpin</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lseparatorLabel">
         <property name="text">
          <string>Lot separator</string>
         </property>
         <property name="buddy">
          <cstring>lseparatorEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="lseparatorEdit">
         <property name="placeholderText">
          <string>Default (&quot;,&quot;)</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <spacer name="docVerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="filesTab">
      <attribute name="icon">
       <iconset theme="document-new">
        <normaloff>.</normaloff>.</iconset>
      </attribute>
      <attribute name="title">
       <string>Import scans</string>
      </attribute>
      <layout class="QGridLayout" name="filesTabLayout">
       <item row="1" column="1">
        <widget class="QSpinBox" name="fileNameSpin">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <spacer name="fileVerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="filePathSpin">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="filePathLabel">
         <property name="text">
          <string>Path index</string>
         </property>
         <property name="buddy">
          <cstring>filePathSpin</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="fileCommentSpin">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>4</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="fileNameLabel">
         <property name="text">
          <string>Document index</string>
         </property>
         <property name="buddy">
          <cstring>fileNameSpin</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="filesLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="fileEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Path to scan files list</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="fileButton">
           <property name="text">
            <string>Open</string>
           </property>
           <property name="icon">
            <iconset theme="document-open">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="fileCommentLabel">
         <property name="text">
          <string>Comments index</string>
         </property>
         <property name="buddy">
          <cstring>fileCommentSpin</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="fileRoleLabel">
         <property name="text">
          <string>Role index</string>
         </property>
         <property name="buddy">
          <cstring>fileRoleSpin</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="fileRoleSpin">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>3</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="dictTab">
      <attribute name="icon">
       <iconset theme="address-book-new">
        <normaloff>.</normaloff>.</iconset>
      </attribute>
      <attribute name="title">
       <string>Import dictionary</string>
      </attribute>
      <layout class="QGridLayout" name="dictTabLayout">
       <item row="1" column="1">
        <widget class="QSpinBox" name="dictIdSpin">
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <spacer name="dictVerticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="dictNameSpin">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout" name="dictLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="dictEdit">
           <property name="readOnly">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string>Path to scan files list</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="dictButton">
           <property name="text">
            <string>Open</string>
           </property>
           <property name="icon">
            <iconset theme="document-open">
             <normaloff>.</normaloff>.</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="dictNameLabel">
         <property name="text">
          <string>Name index</string>
         </property>
         <property name="buddy">
          <cstring>dictNameSpin</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="dictIdLabel">
         <property name="text">
          <string>ID index</string>
         </property>
         <property name="buddy">
          <cstring>dictIdSpin</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>documentEdit</tabstop>
  <tabstop>documentButton</tabstop>
  <tabstop>docNameSpin</tabstop>
  <tabstop>docCommunitySpin</tabstop>
  <tabstop>docPrecinctSpin</tabstop>
  <tabstop>docLootSpin</tabstop>
  <tabstop>docCommentSpin</tabstop>
  <tabstop>docOwnerSpin</tabstop>
  <tabstop>lseparatorEdit</tabstop>
  <tabstop>fileEdit</tabstop>
  <tabstop>fileButton</tabstop>
  <tabstop>fileNameSpin</tabstop>
  <tabstop>filePathSpin</tabstop>
  <tabstop>fileRoleSpin</tabstop>
  <tabstop>fileCommentSpin</tabstop>
  <tabstop>dictEdit</tabstop>
  <tabstop>dictButton</tabstop>
  <tabstop>dictIdSpin</tabstop>
  <tabstop>dictNameSpin</tabstop>
  <tabstop>fseparatorEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ImportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>587</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ImportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>491</x>
     <y>587</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>documentEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ImportDialog</receiver>
   <slot>checkData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ImportDialog</receiver>
   <slot>checkData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>docNameSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportDialog</receiver>
   <slot>checkData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>docCommunitySpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportDialog</receiver>
   <slot>checkData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>docPrecinctSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportDialog</receiver>
   <slot>checkData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>docLootSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportDialog</receiver>
   <slot>checkData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>docCommentSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportDialog</receiver>
   <slot>checkData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>docOwnerSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportDialog</receiver>
   <slot>checkData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fseparatorEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ImportDialog</receiver>
   <slot>checkData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>588</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lseparatorEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ImportDialog</receiver>
   <slot>checkData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>documentButton</sender>
   <signal>clicked()</signal>
   <receiver>ImportDialog</receiver>
   <slot>openDocument()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileButton</sender>
   <signal>clicked()</signal>
   <receiver>ImportDialog</receiver>
   <slot>openFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dictButton</sender>
   <signal>clicked()</signal>
   <receiver>ImportDialog</receiver>
   <slot>openDict()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>687</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dictEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ImportDialog</receiver>
   <slot>checkData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>753</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>currentChanged(int)</signal>
   <receiver>ImportDialog</receiver>
   <slot>checkData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>14</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>776</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileNameSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportDialog</receiver>
   <slot>checkData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>741</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>filePathSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportDialog</receiver>
   <slot>checkData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>415</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>703</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileCommentSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportDialog</receiver>
   <slot>checkData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>483</x>
     <y>163</y>
    </hint>
    <hint type="destinationlabel">
     <x>745</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dictIdSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportDialog</receiver>
   <slot>checkData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>403</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>693</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dictNameSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportDialog</receiver>
   <slot>checkData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>861</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fileRoleSpin</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImportDialog</receiver>
   <slot>checkData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>790</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>checkData()</slot>
  <slot>openDocument()</slot>
  <slot>openFile()</slot>
  <slot>openDict()</slot>
 </slots>
</ui>
